source/learn/getting-started/rails-setup.html.md

```diff
@@ -70,6 +70,15 @@ In Rails environment ROM container is accessible via `ROM.env`:
 ROM.env # returns the container
 ```
 
+In your controllers you can access ROM container simply like this:
+``` ruby
+class HomesController < ApplicationController
+  def show
+    rom #=> returns rom container
```

Comment:
Maybe show this doing something valid in a rails context?  For some reason, I find the blankness of it distracting.
```
@home = rom.relation(:homes).by_id(params[:id]).one
```